controllers/vspherevm_controller.go

```diff
@@ -261,7 +292,35 @@ func (r vmReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.R
 			),
 		)
 
+		// TODO: Implement a fallback mode in case VSphereMachineVirtualMachineProvisionedV1Beta2Condition does not exist
```

Comment:
Wasn't this done above? Or is this talking about implementing a replacement for above?